doc/man3/SSL_CTX_set_options.pod

```diff
@@ -217,6 +217,17 @@ not propose, and servers will not accept the extension.
 Disable all renegotiation in TLSv1.2 and earlier. Do not send HelloRequest
 messages, and ignore renegotiation requests via ClientHello.
 
+=item SSL_OP_IGNORE_UNEXPECTED_EOF
+
+Some TLS implementations do not send close_notify on shutdown. Before OpenSSL
+3.0, SSL_ERROR_SYSCALL with errno=0 was reported in such cases. This
+behaviour contradicts the B<SSL_ERROR_SYSCALL> documentation. Since OpenSSL 3.0
+the returned error is B<SSL_ERROR_SSL> with a meaningful error on the error
+stack. This option gives the B<SSL_ERROR_ZERO_RETURN> error in such cases.
+You should only enable this option if the protocol running over TLS can detect
+a truncation attack itself, and that the application is checking for that
+truncation attack.
```

Comment:
If you have the same text at 2 location, please update them both

Reply:
Done. Somewhen this documentation should be united...